components/docs-chef-io/content/automate/on_prem_builder.md

```diff
@@ -97,7 +103,10 @@ Accept the license with `y`.
 
 #### Deploy Chef Habitat Builder with Chef Automate Auth
 
-To deploy Chef Habitat Builder with the Chef Automate UI and Auth services only, specify the `builder` product on the command line.
+To deploy Chef Habitat Builder on a different host than Chef Automate, specify only the `builder` product on the command line. Chef Automate UI and Auth services only will also be deployed to support authentication. 
```

Comment:
I think this clarifies your intent, but I was not quite sure what that intent was.
```suggestion
To deploy Chef Habitat Builder on a different host than Chef Automate, specify only the `builder` product on the command line. Note that Chef Automate UI and Auth services will also be deployed to support authentication. 
```